Airports/EGBK/Positions.txt

```diff
@@ -1 +1 @@
-EGBK_I_TWR:Sywell Information:122.700:BKIN:E:EGBK:TWR:-:-:0420:0420:N052.18.19.000:W000.47.34.000::
+EGBK_R_TWR:Sywell Radio:122.700:BKIN:E:EGBK:TWR:-:-:0420:0420:N052.18.19.000:W000.47.34.000::
```

Comment:
```suggestion
EGBK_R_TWR:Sywell Radio:122.700:BKAG:E:EGBK:TWR:-:-:0420:0420:N052.18.19.000:W000.47.34.000::
```

Reply:
I always forget something!
Never knew about the IN / AG thing
Will get the changelog for you :D

Reply:
Thanks @Hinshee - I've just amended the changelog slightly as it's an AIRAC-induced change rather than an enhancement. Otherwise, good to go - will just wait on another review

Reply:
No worries